EIPS/eip-1.md

```diff
@@ -173,11 +173,11 @@ The `resolution` header is required for Standards Track EIPs only. It contains a
 
 #### `discussions-to` header
 
-While an EIP is a draft, a `discussions-to` header will indicate the mailing list or URL where the EIP is being discussed. As mentioned above, examples for places to discuss your EIP include [Ethereum topics on Gitter](https://gitter.im/ethereum/topics), an issue in this repo or in a fork of this repo, [Ethereum Magicians](https://ethereum-magicians.org/) (this is suitable for EIPs that may be contentious or have a strong governance aspect), and [Reddit r/ethereum](https://www.reddit.com/r/ethereum/).
+While an EIP is a draft, a `discussions-to` header will indicate the mailing list or URL where the EIP is being discussed. As mentioned above, examples for places to discuss your EIP is [Ethereum Magicians](https://ethereum-magicians.org/) (this is suitable for EIPs that may be contentious or have a strong governance aspect).
 
 No `discussions-to` header is necessary if the EIP is being discussed privately with the author.
```

Comment:
We should remove this line, it is required in all cases.

Reply:
Removed "No `discussions-to` header is necessary if the EIP is being discussed privately with the author."